modules/http/src/test/java/org/glassfish/grizzly/http/HttpResponseParseTest.java

```diff
@@ -124,7 +123,7 @@ public void testDecoder100continueThen200() {
     }
 
     public void testDecoder100continueWithAHeaderThen200() {
-        HttpPacket packet = doTestDecoder("HTTP/1.1 100 Continue\r\nConnection: keep-alive\r\n\r\nHTTP/1.1 200 OK\r\n\r\n", 4096);
+        HttpPacket packet = doTestDecoder("HTTP/1.1 100 Continue\r\nConnection: keep-alive\n\nHTTP/1.1 200 OK\n\n", 4096);
```

Comment:
Just a comment: notice that \n\n is the way grizzly has to identify that those were two responses.